docs/organizations/verify-user-permissions.mdx

```diff
@@ -6,26 +6,16 @@ description: A collection of utility functions and components in order to allow
 # Verify the active user's permissions in an organization
 
 <Callout type="info">
-  The following authorization checks are predicated on a user having an active organization. Without this, they will likely always evaluate to false by default. Clerk's built-in [`<OrganizationSwitcher/>`](/docs/components/organization/organization-switcher) component and `[useOrganizationList()](/docs/references/react/use-organization-list).setActive({ organization: <orgId> })` method are two available options to set an organization as active for a user.
+  The following authorization checks are predicated on a user having an active organization. Without this, they will likely always evaluate to false by default. Clerk's built-in [`<OrganizationSwitcher/>`](/docs/components/organization/organization-switcher) component and[`useOrganizationList().setActive({ organization: <orgId> })`](/docs/references/react/use-organization-list) method are two available options to set an organization as active for a user.
 </Callout>
 
 Clerk provides two kinds of helpers for authorization:
```

Comment:
```suggestion
Clerk enables two broad approaches to authorization:
```

Reply:
@alexisintech This and the next couple suggested changes should reword and reframe the intro to this page. Feel free to change them as you please 👍. I think a thorough cleanup of this page should be part of a separate linear ticket, as I don't want to block this PR for much longer